.taskcluster.yml

```diff
@@ -30,6 +30,14 @@ tasks:
         then: ${event.after}
         else: ${event.release.tag_name}
 
+    push_date_time:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.head.repo.pushed_at}
+      else:
+        $if: 'tasks_for == "github-push"'
+        then: ${event.repository.pushed_at}
+        else: ${event.release.published_at}
```

Comment:
You might need to update `schedule_nightly_task`

Reply:
Same thing about this change. It's just my temporary commit